spec/src/main/asciidoc/examples.asciidoc

```diff
@@ -0,0 +1,92 @@
+//
+// Copyright (c) 2018 Contributors to the Eclipse Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+[[examples]]
+== MicroProfile Concurrency Examples
+
+This page includes some additional examples of spec usage.
```

Comment:
I suppose all these "pages" are put together into one big document, hence maybe using `This section` or `This part` would be more spot on.

Reply:
Updating to "This section"